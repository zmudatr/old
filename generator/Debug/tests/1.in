USTAW V WD
WYPISZ V
ZERUJ V
USTAW W YEYYIu
WYPISZ W
ZERUJ W
USTAW R Vy
WYPISZ R
ZERUJ R
USTAW V ShLYxHVw
WYPISZ V
ZERUJ V
USTAW U l8PJyGC-m
WYPISZ U
ZERUJ U
USTAW U OaFBc19lWk
WYPISZ U
ZERUJ U
USTAW K 3rE
WYPISZ K
ZERUJ K
USTAW B v8P2XDJD
WYPISZ B
ZERUJ B
USTAW C ScXe4lR
WYPISZ C
ZERUJ C
USTAW U vYq1gcgggR
WYPISZ U
ZERUJ U
USTAW A QbX3zV3E6
WYPISZ A
ZERUJ A
USTAW M AED
WYPISZ M
ZERUJ M
USTAW V G
WYPISZ V
ZERUJ V
USTAW L e
WYPISZ L
ZERUJ L
USTAW O x_a6T
WYPISZ O
ZERUJ O
USTAW X p20Sdf1MbK
WYPISZ X
ZERUJ X
USTAW J kcadVBu
WYPISZ J
ZERUJ J
USTAW D 5YDy/YqPDN
WYPISZ D
ZERUJ D
USTAW A *jLdfC9M
WYPISZ A
ZERUJ A
USTAW A +j
WYPISZ A
ZERUJ A
