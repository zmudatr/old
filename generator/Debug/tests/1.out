WD
YEYYIu
Vy
ShLYxHVw
l8PJyGC-m
OaFBc19lWk
3rE
v8P2XDJD
ScXe4lR
vYq1gcgggR
QbX3zV3E6
AED
G
e
x_a6T
p20Sdf1MbK
kcadVBu
5YDy/YqPDN
*jLdfC9M
+j
